### ISSUE TITLE
Store and restaurant Discounts

### ISSUE THREAD

**Issue by OysterRancher** [department-of-veterans-affairs/va-data#13]
Quite a few stores and restaurants offer discounts to active duty, retired service men and women and veterans.  It would be great to have a database where we could go to see who they are.  We could then reward them with our business.   I'll start with a few I know and maybe someone can start a database.  Probably should be categorized by state etc.  Here is my short list
Lowes
Shoppers Food Warehouse
Outback Steak House
Bob Evans

**Comment by BlackThumbEDQ** [comment]
I agree - a database would be awesome. My husband and I frequently use military discounts at hotels & stores - less frequently at restaurants. Just FYI - I always use my my VA health card as picture ID with my credit card rather than ask if they offer a military discount.
Another option to check into is VeteransAdvantage.com - it's not free but the discounts I've used have saved me tons over the last 7 years or so.  And be wary of Lowes - their new military discount id proof makes me somewhat nervous over the security of the info - I don't like providing the date of separation to anybody!
I do have some experience with database design but probably not enough to do this.

**Comment by cluckycluck** [comment]
ID.ME was set up a few years back for this purpose. Initially, it was solely for veterans, but I believe it is now available for teachers and firefighters, as well. Not sure if they've made it more user friendly over the years, but I still receive emails from them, so it should still be available. Hope this helps.

**Comment by walkingeagle** [comment]
ID.ME has come a long way it was first set up just for Vets but now it covers Vets, Police, Firemen, and all the major Homeland alphabet groupes as well as some specialized groupes too. So you do not have to worry about this one.
